packages/builder/src/components/start/CreateAppModal.svelte

```diff
@@ -9,17 +9,53 @@
   import { goto } from "@roxi/routify"
   import { createValidationStore } from "helpers/validation/yup"
   import * as appValidation from "helpers/validation/yup/app"
+  import TemplateCard from "components/common/TemplateCard.svelte"
 
   export let template
 
+  let creating = false
+
   const values = writable({ name: "", url: null })
   const validation = createValidationStore()
   $: validation.check($values)
 
   onMount(async () => {
+    $values.name = resolveAppName(template, $values.name)
+    nameToUrl($values.name)
     await setupValidation()
   })
 
+  $: appUrl = `${window.location.origin}${
```

Comment:
I think this will need an `/app` prefix in here. e.g. https://bb.budibase.app/app/agency%20c%20portal%20test

Reply:
Yes, that's correct. I had a chat with @aptkingston to confirm the expected behaviour regarding the url display here.  A fix has been applied and merged.